inc/views/base/usercp/usergroups.twig

```diff
@@ -0,0 +1,132 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bburl }} - {{ lang.group_memberships }}</title>
+{% endblock head %}
+
+{% block body %}
+    {% if leadinggroups %}
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+            <tr>
+                <td class="thead" colspan="3"><strong>{{ lang.usergroups_leader }}</strong></td>
+            </tr>
+            <tr>
+                <td class="tcat" width="36%"><strong><span class="smalltext">{{ lang.usergroups_usergroup }}</span></strong></td>
+                <td class="tcat" align="center" width="32%"><strong><span class="smalltext">{{ lang.usergroup_members }}</span></strong></td>
+                <td class="tcat" align="center" width="32%"><strong><span class="smalltext">{{ lang.join_requests }}</span></strong></td>
+            </tr>
+            {% for usergroup in leadinggroups %}
+                <tr>
+                    <td><strong>{{ usergroup.title|e }}</strong></td>
```

Comment:
Maybe I'm being daft, but what's the point of the `|e` filter here? We escape by default.

Reply:
My bad, forgot the default behavior 😭